lib/taskana-core/src/main/java/pro/taskana/jobs/JobRunner.java

```diff
@@ -0,0 +1,64 @@
+package pro.taskana.jobs;
```

Comment:
Now we have two classes with the same name impl/jobs/JobRunner.java for tasks and impl/JobRunner.java for jobs table jobs, I think it confusing, we should clarify one of them or maybe integrate Bern's job into  your new JobRunner

Reply:
this will be done with the next pr